libs/blockchain/src/blockchain.service.ts

```diff
@@ -154,6 +155,10 @@ export class BlockchainService extends BlockchainRpcQueryService implements OnAp
   public async payWithCapacity(
     tx: Call | IMethod | string | Uint8Array,
   ): Promise<[SubmittableExtrinsic<'promise'>, HexString]> {
+    const outOfCapacity = await this.checkTxCapacityLimit(this.config.providerId, tx.toString());
+    if (outOfCapacity) {
+      throw new DelayedError();
+    }
```

Comment:
if there isnt enough capacity for tx then throw DelayedError that should be handled at caller end